test/unit_overapproximate.jl

```diff
@@ -129,4 +129,22 @@ for N in [Float64, Float32]
     Y_zonotope = overapproximate(Y, Zonotope) # overapproximate with a zonotope
     @test Y_polygon ⊆ Y_zonotope
     @test !(Y_zonotope ⊆ Y_polygon)
+
+    #Decomposed approximation of lazy linear map of CartesianProductArray
+    i1 = Interval([0, 1])
+    h = Hyperrectangle(low=[3., 3.], high=[5., 5.])
```

Comment:
`h` should not have the same x1 and x2 coordinates to see the difference in the formula.